src/backoffice/media/section.manifests.ts

```diff
@@ -29,15 +29,16 @@ const dashboards: Array<ManifestDashboardCollection> = [
 	},
 ];
 
-const sidebarMenu: ManifestSidebarMenu = {
-	type: 'sidebarMenu',
-	alias: 'Umb.SidebarMenu.Media',
-	name: 'Media Sidebar Menu',
+const sectionSidebarMenu: ManifestSectionSidebarMenu = {
+	type: 'sectionSidebarMenu',
+	alias: 'Umb.SectionSidebarMenu.Media',
+	name: 'Media Section Sidebar Menu',
 	weight: 100,
 	meta: {
 		label: 'Media',
 		sections: [sectionAlias],
+		menu: 'Umb.Menu.Media',
```

Comment:
I think this one might be a bit hard to understand? If I understood correctly its a aias for the menu items to bind to?

Totally unconcidered though, could it be the alias, and there by be eliminated? aka. 'Umb.SectionSidebarMenu.Media'

Or would that be fragile?

Reply:
Ah, I realized this is something that goes into the menu. I'll get in touch.